etc/config/fpc.defaults.properties

```diff
@@ -0,0 +1,19 @@
+# Default settings for FPC
```

Comment:
This is used as the default file if nothing else is found. But for the live server, which resides in amazon, you'll probably want to get the compilers from else where. Check the sister repo, `mattgodbolt/compiler-explorer-image` for how to add compilers there. You'll probably want to open a PR on that repo as well

Reply:
Amazon properties added already. Sorry again for not closing this in time :)